src/prommis/uky/uky_flowsheet.py

```diff
@@ -862,5 +874,782 @@ def display_results(m):
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": R_chiller_accounts,
+            "scaled_param": m.fs.R_chiller.duty,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # TODO: How should this be handled considering there are multiple feeds with varying concentrations?
+    m.fs.feed_input = Var(initialize=500, units=units.ton / units.hr)
+    m.fs.feed_grade = Var(initialize=356.64, units=units.ppm)
+
+    hours_per_shift = 8
+    shifts_per_day = 3
+    operating_days_per_year = 336
+
+    # for convenience
+    m.fs.annual_operating_hours = Param(
+        initialize=hours_per_shift * shifts_per_day * operating_days_per_year,
+        mutable=False,
+        units=units.hours / units.a,
+    )
+
+    # TODO: Replace this number with the sum of flowsheet.fs.roaster.flow_mol_comp_product (need to convert units)
+    m.fs.recovery_rate_per_year = Var(
+        initialize=39.3
+        * units.kg
+        / units.hr
+        * 0.8025  # TREO (total rare earth oxide), 80.25% REE in REO
+        * m.fs.annual_operating_hours,
+        units=units.kg / units.yr,
+    )
+
+    # the land cost is the lease cost, or refining cost of REO produced
+    m.fs.land_cost = Expression(
+        expr=0.303736
+        * 1e-6
+        * getattr(units, "MUSD_" + CE_index_year)
+        / units.ton
+        * units.convert(m.fs.feed_input, to_units=units.ton / units.hr)
+        * hours_per_shift
+        * units.hr
+        * shifts_per_day
+        * units.day**-1
+        * operating_days_per_year
+        * units.day
+    )
+
+    # TODO: Revisit this
+    # dummy reagent with cost of 1 USD/kg for each section
+    reagent_costs = (
+        (  # all USD/year
+            # 302962  # Crushing and Screening
+            # + 0  # Dry Grinding
+            +5767543  # Roasting
+            + 199053595  # Leaching
+            + 152303329  # Rougher Solvent Extraction
+            + 43702016  # Cleaner Solvent Extraction
+            + 7207168  # Solvent Extraction Wash and Saponification
+            + 1233763  # Rare Earth Element Precipiation
+            # + 18684816  # Water Treatment
+        )
+        * units.kg
+        / units.a
+    )
+
+    m.fs.reagents = Var(
+        m.fs.time,
+        initialize=reagent_costs / (m.fs.annual_operating_hours),
+        units=units.kg / units.hr,
+    )
+
+    # TODO: Replace these with values from the flowsheet?
+    m.fs.solid_waste = Var(
+        m.fs.time, initialize=11136 / 24, units=units.ton / units.hr
+    )  # non-hazardous solid waste
+    m.fs.precipitate = Var(
+        m.fs.time, initialize=732 / 24, units=units.ton / units.hr
+    )  # non-hazardous precipitate
+    m.fs.dust_and_volatiles = Var(
+        m.fs.time, initialize=120 / 24, units=units.ton / units.hr
+    )  # dust and volatiles
```

Comment:
I imagine these should be replaced with values from the flowsheet. What counts as solid waste - the amount of solid leaving the purge streams (purge streams are currently 100% liquid)?  Is the precipitate the sum of the solids leaving from the filter press? Dust and volatiles will just be the `flow_mol_comp_dust` stream from the roaster (after converting its units)

Reply:
Solid waste would be anything that would end up in a landfill, so most of your bulk solid that is separated. Precipitate is solid produced from treating wastewater or liquid waste streams, i.e. removing dissolved metal that end up on filter cakes.. As you mentioned, the dust and volatiles are produced by the roasters. The solid waste is definitely the largest output stream - the case study found that the total process yield of feed to product streams gave 92.8% solid waste, 6.1% precipitate waste, 1.0% dust and volatiles waste, and <1% actual REE products.

Reply:
I don't believe there would be any solid waste in this flowsheet based on how things are currently set up. Solid only leaves the system via the roaster or the filter press as a filter cake. Here's what the breakdown looks like for where solids end up: 
![image](https://github.com/prommis/prommis/assets/96305519/bde55881-b47c-47de-8a25-5a78461b89c7)

Reply:
There is the waste solids from the leaching stage filter press.

Reply:
Yea, that is entirely what the precipitate mass flow is comprised of. I didn't count that as waste... I suppose we could count it as waste but then we would have no precipitate mass flow.

Reply:
We have two solid outlet streams. There is the final precipitate leaving the roaster and there is also the left-over solids from the leaching stage.

Reply:
I think the issue here was that the costing package defined 3 distinct solid outlet streams (solid waste, precipitate, and dust & volatiles)

- Dust and volatiles = the dust stream from the roaster
- Solid waste = None???
- Precipitate = filter cake from leaching

I don't think the product REEs from the roaster are represented in any of these 3 streams, but feel free to correct me if I'm mistaken about something @bpaul4

Reply:
For ease of reference, this is how they are currently defined. I suppose I could just get rid of the solid waste stream instead of setting it to zero tho.
![image](https://github.com/prommis/prommis/assets/96305519/225ac6e4-d4ad-4c09-ad35-2cd06775b45e)

Reply:
I believe waste from the roaster can be considered dust & volatiles, waste from any leaching can be considered solid waste, and waste from any precipitation or extraction can be considered precipitates. The solid waste should be most of the incoming solid feed, in terms of mass flow, and the precipitate waste should be much smaller.

Reply:
Ok, then the filter cake from the leaching section needs to be updated so that it's treated as solid waste. There is no waste from precipitation or solvent extraction, so the precipitate waste will become 0 (i.e. solid waste and precipitate will be flipped in the screenshot above)

Reply:
Yes, the solids from leaching is solids waste. The "precipitate stream" is a secondary solid waste coming from treatment of the purge stream of the combined residual solid in the leaching filter cake and the recycled stream from the SX rougher.